configs/typescript-base/package.json

```diff
@@ -1,28 +1,32 @@
 {
-  "name": "@canonical/typescript-config-base",
-  "version": "0.0.1",
-  "description": "Vanilla's Typescript configuration",
-  "main": "tsconfig.json",
-  "author": {
-    "email": "webteam@canonical.com",
-    "name": "Canonical Webteam"
-  },
-  "repository": {
-    "type": "git",
-    "url": "https://github.com/canonical/vanilla-monorepo"
-  },
-  "license": "LGPL-3.0",
-  "bugs": {
-    "url": "https://github.com/canonical/vanilla-monorepo/issues"
-  },
-  "homepage": "https://github.com/canonical/vanilla-monorepo#readme",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "devDependencies": {
-    "typescript": "^5.6.2"
-  },
-  "peerDependencies": {
-    "typescript": "^5.6.2"
-  }
+	"name": "@canonical/typescript-config-base",
+	"version": "0.0.1",
+	"description": "Vanilla's Typescript configuration",
+	"main": "tsconfig.json",
+	"author": {
+		"email": "webteam@canonical.com",
+		"name": "Canonical Webteam"
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/canonical/vanilla-monorepo"
+	},
+	"license": "LGPL-3.0",
+	"bugs": {
+		"url": "https://github.com/canonical/vanilla-monorepo/issues"
+	},
+	"homepage": "https://github.com/canonical/vanilla-monorepo#readme",
+	"scripts": {
+		"test": "echo \"Error: no test specified\" && exit 1",
+		"check": "bun run check:biome",
```

Comment:
Yes, and I envision that this will also be accompanied by "test" scripts at a later date.

For CI purposes, I think it looks very clear if the CI runner uses `bun run test`, which runs all tests *and checks*. So running `bun run test` at the top level calls all check and test scripts.